webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
```

Comment:
Hello @cilfonegabriel, please notice that, currently, your project isn't being executed as expected 😉:
![image](https://user-images.githubusercontent.com/42799579/210434849-75e893ed-5f54-4f50-b4b4-348ab0c3ea71.png)
> This message is being shown after `npm start`.

I would recommend that you don't use static names as your `output` in the `webpack.config.js` file:
![image](https://user-images.githubusercontent.com/42799579/210434698-52b1d3c5-7ced-4556-81b9-b1a223b94829.png)

Instead, try using a dynamic file name:
![image](https://user-images.githubusercontent.com/42799579/210434663-15c9a3ea-bc3d-4d10-a940-2e59317d9cb5.png)

It fixed the problem on my side 😄:
![image](https://user-images.githubusercontent.com/42799579/210435086-68047962-07f1-4315-9241-59167045711b.png)

Cheers and keep coding!